tox.ini

```diff
@@ -116,17 +125,56 @@ commands =
     sphinx-build -b html docs/ docs/_build/html -v -w warnings.txt {posargs}
     python tests/utils/check_warnings.py
 
-
-[testenv:docs-live]
+# build the docs with live-reload, if you are working on the docs only (no theme changes) the best option is to call
+# tox run -e docs-live 
+# this will only watch the docs directory and rebuild the docs when changes are detected
+# if you are working on the theme (HTML, jinja templates, JS, CSS) and the docs, you can call
+# tox run -e docs-live-theme
+[testenv:docs-live{,-theme}]
 description = "Build and serve the documentation with live-reload"
-extras = 
+extras =
     dev
-    i18n
 package = editable
-deps = 
+deps =
     sphinx-theme-builder[cli]@git+https://github.com/pradyunsg/sphinx-theme-builder#egg=d9f620b
 # suppress Py3.11's new "can't debug frozen modules" warning
 set_env = PYDEVD_DISABLE_FILE_VALIDATION=1
-commands = 
-    pybabel compile -d src/pydata_sphinx_theme/locale -D sphinx
-    stb serve docs --open-browser --re-ignore=locale|api|_build|\.jupyterlite\.doit\.db
\ No newline at end of file
+commands =
+    docs-live-theme: stb serve docs --open-browser --re-ignore=locale|api|_build|\.jupyterlite\.doit\.db
+    docs-live: sphinx-autobuild docs/ docs/_build/html --open-browser --re-ignore=locale|api|_build|\.jupyterlite\.doit\.db
```

Comment:
Got annoyed at the `docs live` command rebuilding also the JS/CSS assets when working only in docs so I split it here.